src/main/java/frc/robot/commands/shooter/spinShooter.java

```diff
@@ -0,0 +1,45 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Robot;
+
+public class SpinShooter extends CommandBase {
+
+  private double shooterPercentage = 0.35;
+  public SpinShooter(double setpoint) {
```

Comment:
Add a parameter here for `ControlMode`. 

`public SpinShooter(ControlMode controlMode, double setpoint)`

That way you can use the same command for velocity or percentoutput

Reply:
@jojobean23 we didn't add the ControlMode input to the constructor.